tensorflow/stream_executor/rocm/rocsolver_wrapper.h

```diff
@@ -98,9 +98,7 @@ namespace wrap {
   __macro(rocsolver_cunmqr)                 \
   __macro(rocsolver_zunmqr)                 \
   __macro(rocsolver_cungqr)                 \
-  __macro(rocsolver_zungqr)                 \
-  __macro(rocsolver_cheevd)                 \
-  __macro(rocsolver_zheevd)
```

Comment:
do we not use these with ROCm 4.3.1 either?

Reply:
These don't actually exist in ROCm 4.3.1, I didn't catch this when I reviewed Michael's self_adjoint_op PR.

Reply:
got it